services/QuillLMS/client/app/bundles/Evidence/modules/analytics/index.ts

```diff
@@ -1,21 +1,37 @@
 import { Event } from './event_definitions';
 import Events from './events';
 
+import getParameterByName from '../../helpers/getParameterByName';
```

Comment:
I noticed that you updated this file in the `Diagnostic` modules, too.  Are they actually identical?  Should we move this to `Shared`?

Reply:
yeah good catch, we actually have this same function defined in all of the bundles. it's used in a lot of different places, so I'll make a card to consolidate it into the Shared bundle and replace all of its instances